packages/params/src/forkName.ts

```diff
@@ -93,25 +93,3 @@ export const forkPostElectra = exclude(forkAll, [
 export function isForkPostElectra(fork: ForkName): fork is ForkPostElectra {
   return isForkPostDeneb(fork) && fork !== ForkName.deneb;
 }
-
-/**
- * Aliases exported for compatibility. Types and guards above should be used in
- * places where they are more correct than using the "main feature" from a fork.
- */
-export type ForkLightClient = ForkPostAltair;
```

Comment:
we might wanna keep the aliases as it would be a breaking change to the types package otherwise

Reply:
I thought we only care about breaking changes to the cli API which is why we version them all the same?

Reply:
like `lodestar` application is the product and the internal packages are implementation details

Reply:
the reason why I haven't done the refactoring yet is because we wanted to include that in our 2.0 release, the types package is used by a lot of people, eg. https://github.com/ChainSafe/lodestar/issues/7052 and we should avoid breaking it all the time

in this case it's so cheap for us to keep backward compatibility we might as well do it, can remove the aliase in the 2.0 release

Reply:
Ok.  Added them back to `params`